training/views.py

```diff
@@ -36,15 +37,123 @@ def login(request):
 
 def dashboard(request):
     return render(request, 'app_dashboards/training_dashboard.html')
+    
+def sidenavtest(request):
+    return render(request, 'app_dashboards/sidenavtest.html')
 
 def filter_data(request):
     assessments = Assessment.objects.values('id', 'name')
     trainers = Trainer.objects.values('id', 'name')
     states = State.objects.values('id','state_name')
-    participants = Score.objects.filter(training__assessment__id=1).values_list('participant__id', flat=True).distinct()
-    num_trainings = Training.objects.filter(assessment__id=1).values('date', 'place', 'trainer').distinct().count()
+    participants = Score.objects.filter(training__assessment__id=1).values('participant__id').distinct()
```

Comment:
When assessment will be added, we need to do the required changes since it's not feasible to make it dynamic because of the dependencies.